test/test_force.js

```diff
@@ -131,6 +134,17 @@ function main(args) {
         }
     }
 
+    // Clone spm repo and build it if spm-update requested
+    if (spmUpdate) {
+        var spmRepoDir = utils.cloneRepo(tmpDir, spmRepoUri);
+        utils.runProcessCatchError(`${spmRepoDir}/build_xcframeworks.sh`, 'Building xcframeworks for SPM');
```

Comment:
NB: `build_xcframeworks.sh` will build from `forcedotcom/SalesforceMobileSDK-iOS#dev` (see https://github.com/forcedotcom/SalesforceMobileSDK-iOS-SPM/blob/master/build_xcframeworks.sh#L5).
If we want `spm-update` to be able to specify an alternate fork / branch, we will need an extra command line param for `test_force.js`.